doc/InfiniTimeVision.md

```diff
@@ -0,0 +1,32 @@
+# InfiniTime Vision
+
+The purpose of this document is to steer efforts towards a common goal, and as such should be taken into consideration with all developments.
+
+## InfiniTime
+
+InfiniTime is a community built smartwatch firmware, providing freedom and privacy that's usually insufficient in wearable technology.
+
+InfiniTime is not to be used for medical or other health tracking purposes.
```

Comment:
We don't intend InfiniTime to be used as medical device, because we can't (and don't want) to have to meet such a high standard and responsibility. But everyone can do what he/she wants with InfiniTime.

Or is the "not to be used" part some legal speak?
```suggestion
InfiniTime is not intended to be used for medical or other health tracking purposes.
```

Reply:
Of course everyone's free to do whatever regardless of what we say, but we want no part in that. I'd rather keep the statement strong and clear.

Reply:
Good point!